File: _branchPedro/Front/ProEventos-App/src/app/models/Emprestimo.ts
import { Patrimonio } from "./Patrimonio";
import { Usuario } from "./Usuario";

export interface Emprestimo {

  id: number;
  usuarioId?: number
  usuario?: Usuario[];
  // int? acervoId:
  // Acervo? acervo:
  patrimonioId?: number;
  patrimonio?: Patrimonio[];
  devolvido: boolean;
  dataEmprestimo?: Date;
  dataDevolucao?: Date;
  diasAtraso?: number;
  diasAlocacao?: number;

}

File: _branchPedro/Back/src/ProEventos.Domain/Biblioteca/Emprestimo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain.Biblioteca
{
    public class Emprestimo // PATRIMONIO/USUARIO
    {
        public int Id { get; set; }
        public int? UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }
        public int? AcervoId { get; set; }
        public Acervo? Acervo { get; set; }
        public int? PatrimonioId { get; set; }
        public Patrimonio? Patrimonio { get; set; }
        public bool? Devolvido { get; set; }
        public DateTime? DataEmprestimo { get; set; }
        public DateTime? DataDevolucao { get; set; }
        public int? DiasAtraso { get; set; }
        public int? DiasAlocacao { get; set; }
    }
}